### DISCUSSION TITLE
Downloadhelper no longer asks where to download

### DISCUSSION THREAD

**Discussion by tipietil** [aclap-dev/video-downloadhelper#79] [Help and Q&A] [answered]
### Your question

How do I reactivate it to ask where to download? It worked before it updated, now it just downloads to default folder. Hate to move things around after download.

**Comment by paulrouget** [comment, marked as answer]
That's a bug, sorry about that. Working on it.

See: https://github.com/aclap-dev/video-downloadhelper/discussions/73
